lib/connections/tcp_listen.go

```diff
@@ -64,6 +65,9 @@ func (t *tcpListener) Serve() {
 	}
 	defer listener.Close()
 
+	l.Infof("TCP listener (%v) starting", listener.Addr())
```

Comment:
If you mean the one in Service.createListener that's a debug one so I think it's ok